terraform/deployments/variables/common.tfvars

```diff
@@ -1 +1,13 @@
 dex_github_orgs_teams = [{ name = "alphagov", teams = ["gov-uk", "gov-uk-production-deploy"] }]
+
+gds_office_cidrs = ["213.86.153.211/32",
```

Comment:
optional: is there an existing place we can get this from conveniently (e.g. infra-networking remote state maybe??)